adafruit_lsm303.py

```diff
@@ -177,7 +177,8 @@ def raw_magnetic(self):
         """
         self._read_bytes(self._mag_device, _REG_MAG_OUT_X_H_M, 6, self._BUFFER)
         raw_values = struct.unpack_from('>hhh', self._BUFFER[0:6])
-        return tuple([n >> 4 for n in raw_values])
+        values = tuple([n >> 4 for n in raw_values])
+        return (values[0], values[2], values[1])
 
```

Comment:
for my own education -- would it work just as well to just have:
```
return (raw_values[4],raw_values[6],raw_values[5])
```
and avoid the intermediate variable "values"??
Is that just a matter of style or would it save a few bytes as well? Or am I completely wrong....